src/app/cli/src/graphql.ml

```diff
@@ -498,12 +509,27 @@ module Types = struct
       obj "DeleteWalletPayload" ~fields:(fun _ ->
           [pubkey_field ~resolve:(fun _ key -> Stringable.public_key key)] )
 
-    let reset_trust_status =
-      obj "ResetTrustStatusPayload" ~fields:(fun _ ->
-          [ field "ipAddress" ~typ:(non_null string)
-              ~doc:"An IPv4 or IPv6 address"
+    let trust_status =
+      obj "TrustStatusPayload" ~fields:(fun _ ->
+          let open Trust_system.Peer_status in
+          [ field "trust" ~typ:(non_null string) ~doc:"Trust score"
               ~args:Arg.[]
-              ~resolve:(fun _ ip_addr -> Stringable.ip_address ip_addr) ] )
+              ~resolve:(fun _ {trust; _} -> Printf.sprintf "%0.04f" trust)
+          ; field "banned_status" ~typ:(non_null string) ~doc:"Banned status"
+              ~args:Arg.[]
+              ~resolve:(fun _ {banned; _} -> Stringable.banned_status banned)
```

Comment:
Same here -- this should be a nullable `string` where None corresponds to `Unbanned` and `Banned of time` corresponds to unix epoch timestamp (I believe `Time.to_string` will do this for you).